client/tracks/hic/hic.straw.ts

```diff
@@ -64,6 +64,7 @@ hic.atdev controls dev-shortings
 
 */
 
+const oeOption = 'expected'
```

Comment:
@akanksha-r Please remove this const. It's not needed anymore.

Reply:
ok

Reply:
Done